Mod2/Lessons/Week2/ProductiveStruggle.md

```diff
@@ -0,0 +1,53 @@
+# Productive Struggle
+
+### Warm Up
+Reflect in your notebook on the following:
+* During the first few days of the Mod, when did you feel like you were struggling the most?
+* What signs/behaviors/feelings indicate when you are struggling?
+* What have you done in the past to move forward through struggle?
+
+## Productive Struggle and the Learning Zones
+
+![Learning Zones](/Mod2/Images/Week2/LearningZones.png)
+
+**With a Partner** Review the image above.  Identify where you think 'struggle' happens.
+
+** Instructor Note **: we want students to identify that struggle can (and will) happen in the green and red zones. When you are more in the red, you are in _unproductive_ struggle.  More in the green - _productive_ struggle.
+
+
+## Two Types of Struggle
+
+When working through a challenge - in programming or otherwise.  We often find ourselves struggling to understand the problem, or developing a solution.  This struggle isn't bad, its just part of the process!  But, there are different ways of struggling through a problem - some more productive than others.  We can generally group these behaviors under either **Productive Struggle** or **Unproductive Struggle**.  Productive struggle moves you closer to your goal, while unproductive struggle is stagnant, or can even move you further from your goal.
```

Comment:
I'm pretty sure we already introduced Productive and Unproductive struggle in Mod 0, might be good at some point in the lesson to mention we are coming back to these topics. I do think having the definitions again are definitely good!